src/controller/CHIPDeviceController.cpp

```diff
@@ -632,6 +636,16 @@ void DeviceCommissioner::ReleaseCommissioneeDevice(CommissioneeDeviceProxy * dev
         mSystemState->BleLayer()->CloseAllBleConnections();
     }
 #endif
+
+#if CHIP_DEVICE_CONFIG_ENABLE_NFC_BASED_COMMISSIONING
+    Nfc::NFCReaderTransport * readerTransport = DeviceLayer::Internal::NFCCommissioningMgr().GetNFCReaderTransport();
+    if (readerTransport)
+    {
+        ChipLogProgress(Controller, "Stopping discovery of all NFC tags");
+        readerTransport->StopDiscoveringTags();
```

Comment:
I know this tries to do the same thing as BLE, but it's really not ideal:

1. If the commissionee being released is not actually on the NFC transport (we can have multiple commissionee devices around, sort of!), this will do the wrong thing.  Note that BLE does check for BLE transport.
2. Ideally we would clearly communicate that whatever handle we have to NFC is now released and NFC would shut itself down once all the handles are gone, or something, instead of having this global "nuke everything" setup.

Is there a followup tracking something more reasonable here?